build.gradle

```diff
@@ -3597,9 +3607,11 @@ project(":systemTests") {
         testCompile project(":base").sourceSets.test.output
         testCompile project(":controls").sourceSets.test.output
         testCompile project(":swing").sourceSets.test.output
+        testCompile project(":swt")
+        testCompile name: SWT_FILE_NAME
```

Comment:
I note that this does work for me. So copying the pattern used by `systemTests` into the `swt` project might be all that is needed to get it to work there.

Reply:
I assumed I should add it to the robot package, as it is also a visual test. I've moved it to the swt module

Reply:
I verified that the SWT test works well, when manually enabled in build.gradle and if it runs in isolation. As for putting it in a `robot` sub-package, that seems helpful, but since it won't be enabled anyway, is optional.

We can file a follow-up issue to enable SWT tests again (as long as they aren't enabled by default. My thought is to change the default value of `SWT_TEST` to false, and then enable the tests on platforms other than macOS when the `SWT_TEST` is set to `true` along with `FULL_TEST`).